todo_app/src/trello_items.py

```diff
@@ -0,0 +1,52 @@
+import os
+from dataclasses import dataclass
+
+import requests
+
+BASE_URL = "https://api.trello.com/1/"
+
+HEADERS = {"Accept": "application/json"}
+
+BASE_QUERY = {"key": os.getenv("TRELLO_API_KEY"), "token": os.getenv("TRELLO_TOKEN")}
+
+BOARD_ID = os.environ["TRELLO_BOARD_ID"]
```

Comment:
This works nicely here, but pre-loading these may bite you in the next module so worth keeping an eye out for it. 

The reason it could be a problem is that these lines of code run when this file is first _imported_. That works perfectly right now, but:
* When we introduce tests we'll sometimes want to do some setup before we start the app
* This can include loading fake/test environment variables
* But these variables will already have had their values assigned by the time we get to our setup, so we won't affect them

We can fix that by messing with our import order carefully, but we typically end up with very fragile code, so a better solution is one of:
* The easiest right here, just load the environment variables when they're needed, e.g. in `get_items`:
```python
    BASE_QUERY = {"key": os.getenv("TRELLO_API_KEY"), "token": os.getenv("TRELLO_TOKEN")}
    BOARD_ID = os.environ["TRELLO_BOARD_ID"]
    response = requests.get(
        BASE_URL + f"boards/{BOARD_ID}/cards",
        headers=HEADERS,
        params=BASE_QUERY,
    )
```
* Normally I'd be lazy and go with the above here, but on a real codebase we'd probably prefer a more classic OOP solution and your approach for the list ids might justify this here since making an extra request (or even one per-item!) could slow things down in a relevant way; for example we could wrap these functions in a class, and load the environment variables & handle the initialisation of the lists in a constructor, e.g. something like:
```python
class TrelloService():

    def __init__(self):
        self.base_query = {"key": os.getenv("TRELLO_API_KEY"), "token": os.getenv("TRELLO_TOKEN")}
        ...
        self.list_ids_to_name = {v: k for k, v in list_names_to_id.items()}

    def get_items(self):
        response = requests.get(
            BASE_URL + f"boards/{self.board_id}/cards",
            headers=HEADERS,
            params=self.base_query,
        )

        return [Item.from_trello(card, self.list_ids_to_name(card["idList"])) for card in response.json()]
        ...
```

(If you want any extra context about this, see the first half of this module's group call!)